tests/integration/kubernetes/k8s-measured-rootfs.bats

```diff
@@ -45,10 +42,9 @@ setup() {
 	echo "Pod $pod_config file:"
 	cat $pod_config
 
-	assert_pod_fail "$pod_config"
+	kubectl apply -f $pod_config
 
-	assert_logs_contain "$node" kata "$node_start_time" \
-		'verity: .* metadata block .* is corrupted'
+	waitForProcess "60" "3" "exec_host $node journalctl -t kata | grep \"verity: .* metadata block .* is corrupted\""
```

Comment:
Is there any reason why we don't use `assert_pod_fail` and `assert_logs_contain`?

Reply:
There is!  The pod actually never fails, but rather keeps it's status as "Container Creating".
Maybe that's a bug on how things are handled, but we can figure that out after having this one re-enabled.